open_event/api/events.py

```diff
@@ -0,0 +1,41 @@
+from flask.ext.restplus import Resource, Namespace, fields
+
+from open_event.models.event import Event as EventModel
+from .helpers import get_object_list, get_object_or_404
+
+api = Namespace('events', description='Events')
+
+_event = api.model('Event', {
+    'id': fields.Integer(required=True),
+    'name': fields.String,
+    'email': fields.String,
+    'color': fields.String,
+    'logo': fields.String,
+    'start_time': fields.DateTime,
+    'end_time': fields.DateTime,
+    'latitude': fields.Float,
+    'longitude': fields.Float,
+    'slogan': fields.String,
+    'url': fields.String,
+    'location_name': fields.String,
+})
+
+
+@api.route('/<int:event_id>')
+@api.param('event_id')
+@api.response(404, 'Event not found')
+class Event(Resource):
+    @api.doc('get_event')
+    @api.marshal_with(_event)
+    def get(self, event_id):
+        """Fetch an event given its id"""
+        return get_object_or_404(EventModel, event_id)
+
+
+@api.route('/')
+class EventList(Resource):
+    @api.doc('list_events')
+    @api.marshal_list_with(_event)
+    def get(self):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2620995756/issues/source?bid=3233864&fileBranchId=3314204#l39)